content/develop/connect/insight/_index.md

```diff
@@ -159,6 +159,9 @@ These are the locations on supported platforms:
 You can install Redis Insight on operating systems that are not officially supported, but it may not behave as expected.
 {{< /note >}}
 
+## Redis Insight API (only for Docker)
+http://localhost:5540/api/docs
```

Comment:
It certainly makes sense as it is but it might help the reader if you add a brief explanation (and it's also a good place to have a link to the Docker install page):
```suggestion
## Redis Insight API (only for Docker)

If you are running Redis Insight from [Docker]({{< relref "/operate/redisinsight/install/install-on-docker" >}}),
you can access the API from `http://localhost:5540/api/docs`.
```